app/views/test/push_notification/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= content_for(:meta_refresh) { '15' } %>
+
+<h1>Push Notification Events</h1>
+
+<div class="card card-wide">
+  <div class="padding-4">
+    <%= button_to 'Clear events', test_push_notification_path, method: :delete, class: 'usa-button' %>
+    <div class="grid-row grid-gap margin-top-2">
+      <h2>Events</h2>
+
+      <table class="usa-table">
+        <thead>
+          <tr>
+            <th scope="col">Push Notification URL</th>
+            <th scope="col">Payload</th>
+            <th scope="col">JWT</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @events.each do |event| %>
+            <tr>
+              <th scope="row"><%= event[:url] %></th>
+              <td style="overflow-wrap:anywhere"><%= event[:payload].to_json %></td>
```

Comment:
what if we pretty formatted?

```suggestion
              <td style="overflow-wrap:anywhere"><pre><code><%= JSON.pretty_generate(event[:payload]) %></code></pre></td>
```

Reply:
preformatted makes it wide, but it's the most important content so I'm ok with it

![image](https://user-images.githubusercontent.com/1430443/114903426-6bb01580-9ddc-11eb-9dd2-d94a9063222f.png)

Reply:
Maybe instead of a table, update this to be rows of divs?

```
<div>url</div>
<div>JSON blob</div>
<div>raw JWT</div>
```

Reply:
good idea. Revised:
![image](https://user-images.githubusercontent.com/1430443/114916117-095e1180-9dea-11eb-9cca-b0a8f70d873c.png)